src/testcases/run_in_band/ibc_transfer.test.ts

```diff
@@ -215,17 +227,22 @@ describe('Neutron / IBC transfer', () => {
           IBC_RELAYER_NEUTRON_ADDRESS,
           NEUTRON_DENOM,
         );
-        const resBalance =
-          parseInt(balance.amount, 10) - 2333 * 2 - relayerBalance;
-        expect(resBalance).toBeLessThan(5); // it may differ by about 1-2 because of the gas fee
+        const afterBalance = +balance.amount;
+        // two transactions inside the contract means we pay fee = ackFee * 2
+        const resBalance = afterBalance - beforeRelayerBalance - ackFee * 2;
+        expect(resBalance).toBeWithin(-4000, 0); // it may differ by about 3-4k because of the gas fee
       });
       test('contract should be refunded', async () => {
```

Comment:
I suppose it's a mistake in the test name, isn't it?

```suggestion
      test('contract should be charged the fees', async () => {
```